packages/frontend/src/components/screens/MainScreen/MainScreen.tsx

```diff
@@ -161,15 +161,23 @@ export default function MainScreen({ accountId }: Props) {
       null,
       null
     )
-    mediaIds.reverse() // newest first
     const mediaLoadResult = await BackendRemote.rpc.getMessages(
       accountId,
       mediaIds.slice(0, maxIcons)
     )
-    const lastMessages = Object.values(mediaLoadResult).filter(
-      result => result.kind === 'message'
-    )
-    setLastWebxdcApps(lastMessages.reverse()) // show newest first
+    const lastUpdatedApps = mediaIds
+      .reverse()
```

Comment:
I'd prefer the original way, of simply mutating the array after getting it, so that it's harder to accidentally mutate the array another time in the future.